java/ql/src/semmle/code/java/frameworks/Networking.qll

```diff
@@ -129,3 +129,13 @@ class UrlOpenConnectionMethod extends Method {
     this.getName() = "openConnection"
   }
 }
+
+/**
+ * A string matching private host names of IPv4 and IPv6, which only matches the host portion therefore checking for port is not necessary.
```

Comment:
Could you add some examples of strings matched by the regex in this qldoc? The regex is quite hard to read on its own, so a few examples would help, I think.

Reply:
Added